certora/specs/SocialRecoveryModule.spec

```diff
@@ -402,3 +403,15 @@ rule cancelRecoveryDoesNotAffectOtherWallet(env e, address otherWallet) {
         otherRequestBefore.newOwners[i] == otherRequestAfter.newOwners[i] &&
         otherWalletNonceBefore == currentContract.walletsNonces[otherWallet];
 }
+
+rule cantFinalizeRecoveryBeforeDelayPeriod(env e) {
```

Comment:
nit: As we check the success scenario, maybe use a different name like `canFinalizeRecoveryOnlyAfterDelayPeriod` or something similar.

Reply:
Also, some comments to describe the rule.

Reply:
Perhaps an unpopular opinion, but to me, most of the comments in the specs seem excessive. A good comment should explain "How?" and "Why?", not "What?" and duplicate what the code already says. Most of the comments in the specs just rephrase variable names/code in plain English sentences, and that doesn't provide any value.

Reply:
An example of such a comment:
```
// This rule verifies that the guardian can always initiate recovery considering some ideal conditions.
rule confirmRecoveryCanAlwaysBeInitiatedByGuardian(env e, address guardian, address[] newOwners, uint256 newThreshold, bool execute) {
```

What would you have missed without this comment?

Reply:
One more:
```
// Cancelling recovery for a wallet does not affect other wallets
rule cancelRecoveryDoesNotAffectOtherWallet(env e, address otherWallet) {
```

Reply:
This is a good comment:
```
// This integrity rule verifies that if the revokeGuardianWithThreshold(...) executes, then ensure that:
// - the Social Recovery Module is enabled
// - the caller to the Module has to be the Safe Contract
// - the guardian is revoked from the guardian list
// - the linked list integrity remains,
// - and no other account (guardian or not) is affected.
rule revokeGuardiansWorksAsExpected(env e, address guardian, address prevGuardian, uint256 threshold, address otherAccount) {
```

But I don't think it's possible to come up with something similar for my rule because it's dead simple

Reply:
Makes sense 👍🏾 Sometimes the code is too simple to explain things 😅